pygerduty/events.py

```diff
@@ -22,9 +22,9 @@ def __str__(self):
 
 
 class Events(object):
-    def __init__(self, service_key, requester):
+    def __init__(self, service_key):
```

Comment:
I think it's still valuable to allow a Requester to be passed. Maybe something like 

```python
    def __init__(self, service_key, requester=None):
        self.service_key = service_key
        if requester is None:
            self.requester = pygerduty.common.Requester()
        else:
            self.requester = requester
```